src/pages/workspaces/workspace/Data.js

```diff
@@ -1002,6 +1047,26 @@ const WorkspaceData = _.flow(
               }
             }, [`${type} (${typeDetails.count})`])
           }, sortedEntityPairs),
+          div({ style: Style.navList.heading }, ['Snapshots']),
+          _.map(([snapshotName, { id: snapshotId, entityMetadata: snapshotTables }]) => {
```

Comment:
Make sure you're looking at the errors in the browser console when things are being rendered. Here, React is complaining that it's being given an array to render as a child (that is, within the children array of this div) without keys on its members. Whenever we're giving React a list of things to render within children, each should be given a unique `key` prop, so React can keep track of them. [See here](https://reactjs.org/docs/lists-and-keys.html) for more.